README.md

```diff
@@ -1 +1,26 @@
 # FINDOLOGIC Shopware 6 Plugin Override
+
+This is a basic plugin skeleton which may be used to customize
+[FINDOLOGIC's Shopware 6 plugin](#).
+
+## Requirements
+Note that in order to run this plugin you will need to install
+[FINDOLOGIC Search](#) first.
+
+## Installation
```

Comment:
You've spent some time writing this installation guide, but I think it is not necessary, and if it was, this would be the wrong place. Please remove it.

Reply:
ah yes, I think I got confused with the actual plugin as I had that opened as well for reference.